DeepSpeech.py

```diff
@@ -1121,6 +1129,7 @@ def start_coordination(self, data_sets, step=0):
             # We only have to train, if we are told so and are not at the target epoch yet
             self._train = FLAGS.train and self._target_epoch > self._epoch
             self._test = FLAGS.test
+            self._dev = FLAGS.dev
```

Comment:
Where is `FLAGS.dev` defined? What's its meaning in this context? If validation is to be done? But that's flagged by `FLAGS.validation_step > 0`.